mmv1/third_party/terraform/website/docs/r/sql_user.html.markdown

```diff
@@ -74,6 +74,34 @@ resource "google_sql_user" "iam_service_account_user" {
 }
 ```
 
+Example using [Cloud SQL IAM Group authentication](https://cloud.google.com/sql/docs/mysql/iam-authentication#iam-group-auth).
+
+```hcl
+resource "random_id" "db_name_suffix" {
+  byte_length = 4
+}
+
+resource "google_sql_database_instance" "main" {
+  name             = "main-instance-${random_id.db_name_suffix.hex}"
+  database_version = "MYSQL_8_0"
+
+  settings {
+    tier = "db-f1-micro"
+
+    database_flags {
+      name  = "cloudsql.iam_authentication"
+      value = "on"
+    }
+  }
+}
+
+resource "google_sql_user" "iam_group_user" {
+  name     = "iam_group@example.com"
+  instance = google_sql_database_instance.main.name
+  type     = "CLOUD_IAM_GROUP"
```

Comment:
Can you also add the new value to the `type` field doc here?
https://github.com/GoogleCloudPlatform/magic-modules/blob/417f6cf48584d72b371e4533ed33b435659f0381/mmv1/third_party/terraform/website/docs/r/sql_user.html.markdown?plain=1#L120